www/delivery_dev/ck.php

```diff
@@ -84,12 +84,12 @@
         // without recording the click or performing redirection
         if (isset($GLOBALS['conf']['logging']['blockInactiveBannerClicks'])) {
             $aAdInfo = MAX_cacheGetAd($adId[$i]);
-            if ($aAdInfo['status'] != 0 || $aAdInfo['campaign_status'] != 0) {
+            if ($aAdInfo['status'] != OA_ENTITY_STATUS_RUNNING || $aAdInfo['campaign_status'] != OA_ENTITY_STATUS_RUNNING) {
```

Comment:
Thanks! I thought there was probably one of these, but couldn't find it in the time I had to work on the issue yesterday. :-)